packages/emails/src/components/WhoInfo.tsx

```diff
@@ -35,6 +35,14 @@ export function WhoInfo(props: { calEvent: CalendarEvent; t: TFunction }) {
               email={attendee.email}
             />
           ))}
+          {props.calEvent.team?.members.map((member) => (
```

Comment:
I think we should show team members before attendees so that it is shown along with the organizer who is also a team member. All team members would be together.

Reply:
https://github.com/calcom/cal.com/pull/7207/commits/25a3d97751ca3cbc0b6711aba0024b955404f7f9